test/test_cases/suffix_comments_good.ex

```diff
@@ -0,0 +1,23 @@
+def test() do
+  # comment 1
+  if true do
+    # comment 2
+    if false do
+      nil
+      # comment 3
```

Comment:
I know this is a slightly contrived case, since it also inserts the missing `nil`. But I feel like it could be nicer to have the comment before `nil` - it seems to me like a more natural choice - the comment most often goes above the code it annotates.